README.md

```diff
@@ -160,6 +161,9 @@ See the Hootenanny User Guide for additional usage examples and details on comma
     # Conflate, adding data from the second map to output that does not conflict with data in 
     # the first map
     hoot conflate --differential input1.osm input2.osm output.osm
+
+    # Conflate only restaurants (see the User Guide for more filter examples)
+    hoot conflate -D conflate.tag.filter="{ \"must\": [ { \"tag\": \"amenity=restaurant\" } ] }" input1.osm input2.osm output.osm
```

Comment:
This is all great stuff but the interface on the command line might be hard to deal with.  All of the `\"` in there and a min-ified JSON makes it harder to read and produce.  Maybe adding the option to pass in a JSON file by filename would make things easier for the user to understand and produce.  Just a suggestion.  Don't let that hold up this PR.
```
hoot conflate -D conflate.tag.filter=restaurant_filter.json input1.osm input2.osm output.osm
```
Might be a little easier and more extensible.  What do you think?

Reply:
I think having the option of passing a JSON file is a good idea. 
We are going to have to get the filter from the UI some how. I'm not sure how it will go from browser -> services -> command line.

Reply:
I agree.  I'll add a file input option for the command line.  The inline JSON is really only useful for the services and trivial examples.  The UI passes in JSON through services to core currently for some translation config options.